WordPressLoginFlow/build.gradle

```diff
@@ -59,17 +66,9 @@ dependencies {
 
     api 'com.google.android.gms:play-services-auth:18.1.0'
 
-    // Share FluxC version from host project if defined, otherwise fallback to default
-    if (project.hasProperty("fluxCVersion")) {
-        implementation("com.github.wordpress-mobile.WordPress-FluxC-Android:fluxc:$fluxCVersion") {
-            exclude group: "com.android.support"
-            exclude group: "org.wordpress", module: "utils"
-        }
-    } else {
-        implementation("com.github.wordpress-mobile.WordPress-FluxC-Android:fluxc:1.17.0") {
-            exclude group: "com.android.support"
-            exclude group: "org.wordpress", module: "utils"
-        }
+    implementation("org.wordpress:fluxc:1.23.0") {
```

Comment:
Just to confirm my understanding: how will that line evolve as new versions of FluxC start to get released?

I saw in the PR description your explanation about the client's version taking priority over this transitive dependency, so iiuc when we'll release `1.24.0` and even higher versions like `1.37.42`, clients will just update their `fluxCVersion` in the client's `build.gradle` but not touch this line from the Login-Flow lib, which will stay at `1.23.0` for a long time… unless if this Login-Flow lib itself requires new features and APIs from FluxC to implement a new feature in LoginFlow, right?

> if the dependency declaration matches. Meaning the group and module information are the same for each dependency.

Does that mean that even if the client uses `2.3.4` (but still uses the same group and module), that `2.3.4` will be used… even if it shouldn't be compatible semantic-versioning-wise (as in semver world, going from 1.x.y to 2.x.y is a breaking change)? _I remember we discussed about Gradle dependency version resolution and how semver was not as prominent in Gradle as in other dependency managers in other languages, but just wanted to confirm I recalled that correctly_

Reply:
Thanks for the explanation, all clear now! 👌